docs/data/system/styled/styled.md

```diff
@@ -40,6 +40,9 @@ It aims to solve the same problem, but also provides the following benefits:
    - `options.skipSx` (_bool_ [optional]): Disables the `sx` prop on the component.
    - The other keys are forwarded to the `options` argument of emotion's [`styled([Component], [options])`](https://emotion.sh/docs/styled).
 
+3. `styles` (_object | `({ ...props, theme }) => object`_ [optional]): A styles object or a function returning a styles object.
```

Comment:
```suggestion
3. `styles` (_object | `({ ...props, theme }) => object`): A styles object or a function returning a styles object.
```
As far as I remember, the `styles` is required. Could you again if the code below work?

```js
const Foo = styled('div')();
```